CODEOWNERS

```
@@ -1,5 +1,40 @@
 # Code in the `rasa.shared` package is potentially re-used by downstream dependencies
 # such as Rasa X. Hence, changes within this package require double checking.
-/rasa/shared/ @RasaHQ/enable-squad
 /docs/docs/prototype-an-assistant.mdx @ricwo @alwx
 /.github/workflows/ @RasaHQ/infrastructure
+*.py @RasaHQ/enable-squad
```

I think we've to limit this to the `rasa/` dir (otherwise we also get assigned for tests). According to the docs it would be something like this:
```suggestion
rasa/**.py @RasaHQ/enable-squad
```